react-multiple-pages/my-app/src/App.tsx

```diff
@@ -1,27 +1,19 @@
-import { useState } from 'react';
-import { About } from './pages/About';
-import { Dashboard } from './pages/Dashboard';
+import { Route, Routes } from 'react-router-dom';
 import { Header } from './components/Header';
-import { NotFound } from './pages/NotFound';
+import { Dashboard } from './pages/Dashboard';
+import { About } from './pages/About';
 import { Details } from './pages/Details';
-import { Route, Routes } from 'react-router-dom';
+import { NotFound } from './pages/NotFound';
 
 export function App() {
-  // const [page, setPage] = useState('dashboard');
-  const [itemId, setItemId] = useState<number>();
-
   return (
-    <>
-      <Header />
-      <Routes>
-        <Route
-          path="/"
-          element={<Dashboard onDetails={(id) => setItemId(id)} />}
-        />
+    <Routes>
+      <Route path="/" element={<Header />}>
+        <Route path="index" element={<Dashboard />} />
```

Comment:
You want to remove `path="index"` and replace it with just `index`. The `index` prop tells React Router that this is the default path.
```
<Route index element={<Dashboard />} />
```